### ISSUE TITLE
LG A1 48 not able to turn off anymore after LG update 03.30.14. turn on still fine.

### ISSUE THREAD

**Issue by gruan8886** [JPersson77/LGTVCompanion#103]
it all happened right after LG software update.  TV will not turn off anymore. also not able to do idle.

**Comment by JPersson77** [comment]
I think it is possible LG changed something. Will investigate

**Comment by RoofDefenseSystems** [comment]
Yes, I recently received Software Version 03.30.14 I am currently installing the dev build to try and get you more info.

**Comment by RoofDefenseSystems** [comment]
C:\ProgramData\LGTV Companion

**Comment by RoofDefenseSystems** [comment]
[Log.txt](https://github.com/JPersson77/LGTVCompanion/files/10440792/Log.txt)

**Comment by JPersson77** [comment]
thanks! it really looks like LG has taken action to close down this port. in the recent firmware Will investigate more

**Comment by gruan8886** [comment]
Thanks for the effort. Great program. Hope you can fix it soon. 

 �

**Comment by JPersson77** [comment]
A fix has been implemented and will be in the next release

**Comment by gruan8886** [comment]
great news. Thanks alot .

**Comment by pvprodk** [comment]
Same behaviour on C2 42 with software version 03.30.16. Excited to see if next release fixes for this software version as well.

**Comment by JPersson77** [comment]
New version is up (v2) so I'm closing this issue. https://github.com/JPersson77/LGTVCompanion/releases
